NewHorizons/Builder/Atmosphere/CloudsBuilder.cs

```diff
@@ -90,7 +90,7 @@ public static void Make(GameObject planetGO, Sector sector, AtmosphereModule atm
 
             try
             {
-                fluidType = (FluidVolume.Type)Enum.Parse(typeof(FluidVolume.Type), Enum.GetName(typeof(CloudFluidType), atmo.clouds.fluidType).ToUpper());
```

Comment:
This conversion code is shared between CloudBuilder, RingBuilder, and TornadoBuilder. We should make it all a single function called in these three places that converts `FluidType` to `FluidVolume.Type`